commands/System/eval.js

```diff
@@ -5,6 +5,7 @@ exports.run = async (client, msg, [code]) => {
   try {
     let evaled = eval(code);
     if (typeof evaled !== "string") evaled = inspect(evaled);
+    if (evaled instanceof Promise) evaled = await evaled;
```

Comment:
Shouldn't be this before inspecting it? So if the Promise returns an object, it'll be inspected.